features/networking/service.feature

```diff
@@ -786,23 +786,23 @@ Feature: Service related networking scenarios
     Given I use the "<%= cb.masters[1].name %>" node
     #It should work because its external traffic from another node and destination node has a backend pod on it (ETP=local respected)
     When I run commands on the host:
-      | curl --connect-timeout 5 <%= cb.hostip %>:<%= cb.port %> |
+      | curl --connect-timeout 5 [<%= cb.hostip %>]:<%= cb.port %>|
```

Comment:
Also made `externalTrafficPolicy Local` case IPv6 compatible by just including the braces.

Reply:
ah, good to know braces can work on both v4 and v6